test/helpers/index.js

```diff
@@ -0,0 +1,2 @@
+const chai = require('chai');
+global.expect = chai.expect;
```

Comment:
No, the alternative is to put those two lines in the top of every test file, but given that test suites are already littered with globals ('describe', 'it', 'before', etc) then adding one more is a cleaner approach that repeating that same code block all over the place.